Modules/Synchronization/test/Synchronization.Application.Tests/Tests/IntegrationEvents/IdentityDeletionProcessStartedIntegrationEventHandlerTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.EventBus;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Synchronization.Application.Infrastructure;
+using Backbone.Modules.Synchronization.Application.IntegrationEvents.Incoming.IdentityDeletionProcessStarted;
+using Backbone.Modules.Synchronization.Application.IntegrationEvents.Outgoing;
+using Backbone.Modules.Synchronization.Domain.Entities.Sync;
+using FakeItEasy;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Backbone.Modules.Synchronization.Application.Tests.Tests.IntegrationEvents;
+
+public class IdentityDeletionProcessStartedIntegrationEventHandlerTests
+{
+    [Fact]
+    public async Task Creates_external_events_after_consuming_integration_event()
```

Comment:
It only creates one event, right?

```suggestion
    public async Task Creates_an_external_event()
```